tracer/test/Datadog.Trace.Tests/Tagging/TagsListTests.cs

```diff
@@ -157,6 +150,52 @@ public void Serialization(bool topLevelSpan)
             }
         }
 
+        [Fact]
+        public void Serialize_ManualSpans_LanguageTag()
+        {
+            // manual spans use CommonTags
+            var span = new Span(new SpanContext(42, 41), DateTimeOffset.UtcNow);
+            var deserializedSpan = SerializeSpan(span);
+
+            deserializedSpan.Tags[Tags.Language].Should().Be(TracerConstants.Language);
+        }
+
+        [Theory]
+        [InlineData(SpanKinds.Client, null)]
+        [InlineData(SpanKinds.Server, TracerConstants.Language)]
+        [InlineData(SpanKinds.Producer, TracerConstants.Language)]
+        [InlineData(SpanKinds.Consumer, TracerConstants.Language)]
+        [InlineData("other", TracerConstants.Language)]
+        public void Serialize_InstrumentationTags_LanguageTag(string spanKind, string expectedLanguage)
+        {
+            var tags = new Mock<InstrumentationTags>();
+            tags.Setup(t => t.SpanKind).Returns(spanKind);
```

Comment:
nit: I would create a stub and just set the span rather than using a Mock, but that's maybe just me :)

Reply:
I don't feel strongly either way, but this seemed like less work, especially since we would use the stub only once.

Reply:
That's what I've said in the past, and then been bitten by weird failures in apparently unrelated tests 😉 But I really am ambivalent here 😄